DataFormats/PatCandidates/interface/Tau.h

```diff
@@ -38,6 +38,7 @@ namespace pat {
   class Tau;
   typedef std::vector<Tau>              TauCollection; 
   typedef edm::Ref<TauCollection>       TauRef; 
+  typedef edm::RefProd<TauCollection>	TauRefProd;
```

Comment:
The first instance where it is needed is here: https://github.com/anehrkor/cmssw/blob/runMVAIsolationOnMiniAOD_81X/PhysicsTools/PatAlgos/plugins/PATTauDiscriminationByMVAIsolationRun2.cc#L177

Then, it is also used in https://github.com/anehrkor/cmssw/blob/runMVAIsolationOnMiniAOD_81X/DataFormats/PatCandidates/interface/PATTauDiscriminator.h .

I tried to clone as much as possible from the PFTau counterparts of these classes. Maybe this can be done differently but I don't know how.